views/administration/forms/form-invitation-teacher.hbs

```diff
@@ -1,6 +1,10 @@
 <input name="schoolId" type="hidden" data-force-value="true" value="{{currentSchool}}" />
 <div class="form-group">
-    <label for="invitation-link">Um in die Schul-Cloud einzuladen, verteile folgenden Link an Lehrer.</label>
-    <p>Bevor die Nutzung der Schul-Cloud möglich ist, müssen die Nutzer den elektronischen Einwilligungsprozess durchlaufen haben.</p>
+    <label for="invitation-link">Um in die Schul-Cloud einzuladen, verteile folgenden Link an: </label>
+    <p>
+        <b>Schüler unter 18 Jahre:</b> die Eltern <br/>
```

Comment:
@konstantin-sc Is this correct? https://ticketsystem.schul-cloud.org/browse/SC-643

Reply:
@Parnswir Tickettext alt. Lass uns Montag klären, was schon online ist von den Consentänderungen